vms/shift/services.py

```diff
@@ -331,10 +331,11 @@ def get_shifts_with_open_slots_for_volunteer(j_id, v_id):
     """
     shift_list_by_date = get_shifts_ordered_by_date(j_id)
     shift_list = []
+    today_date= datetime.date.today() + datetime.timedelta(days=1)
 

```

Comment:
Yes @tapasweni-pathak 
Either I should have used < with timedelta 
or <= without timedelta.
I didn't realize that one.
Will make appropriate changes.